frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -1132,9 +1132,9 @@ const loadingDashCards = handleActions(
     [FETCH_DASHBOARD]: {
       next: (state, { payload }) => ({
         ...state,
-        dashcardIds: Object.values(payload.entities.dashcard || {}).map(
-          dc => dc.id,
-        ),
+        dashcardIds: Object.values(payload.entities.dashcard || {})
+          .filter(dc => dc.card.display !== "text")
```

Comment:
Would using `isVirtualDashCard` would actually be better here?

Reply:
Ah didn't realize we had that! I'll swap that in for this and my fix for #11929.

Reply:
Actually the fix for #11929 only had a card not a dashcard so I left as-is. I updated this spot though.